apis/infix-en_GB/atrium-api-infix-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/pathAssertions.kt

```diff
@@ -295,3 +297,52 @@ val <T : Path> Expect<T>.extension: Expect<String>
  */
 infix fun <T : Path> Expect<T>.extension(assertionCreator: Expect<String>.() -> Unit): Expect<T> =
     ExpectImpl.path.extension(this).addToInitial(assertionCreator)
+
+/**
+ * Expects that the subject of the assertion (a [Path]) has the same textual content
+ * as [targetPath].
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.13.0
+ */
+infix fun <T : Path> Expect<T>.hasSameTextualContentAsDefaultArgs(
+    targetPath: Path
+): Expect<T> = addAssertion(ExpectImpl.path.hasSameTextualContentAs(this, targetPath, Charsets.UTF_8, Charsets.UTF_8))
+
+/**
+ * Expects that the subject of the assertion (a [Path]) has the same textual content
+ * as path in [encodingWithCreator].
+ *
+ *  Use the function `withEncoding(Path, Charset, Charset) { ... }` to create a [EncodingWithCreator].
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.13.0
+ */
+infix fun <T : Path> Expect<T>.hasSameTextualContentAs(
+    encodingWithCreator: EncodingWithCreator<T>
+): Expect<T> = addAssertion(ExpectImpl.path.hasSameTextualContentAs(this, encodingWithCreator.path,
```

Comment:
please rebase on master and use 
```
_logicAppend {
    hasSameTextualContentAs(
        encodingWithCreator.path, encodingWithCreator.sourceCharset, encodingWithCreator.targetCharset
    )
}
```

Reply:
_logicAppend isn't accessible from current module. can I add dependency for module 'logic-common'?

Reply:
It will be as soon as you rebase on master